### DISCUSSION TITLE
Regarding author settings in the post

### DISCUSSION THREAD

**Discussion by zhaoyiluo** [HEIGE-PCloud/DoIt#136] [Q&A] [answered]
Hi developers,

I'm at the [author settings](https://hugodoit.pages.dev/theme-documentation-content/#author-setup) guidance step.

It seems that the author's name is not allowed to include white space, otherwise, the redirection will cause an error when you click the author name in one of the posts.

`http://localhost:1313/authors/alice%20wang`

I think it's a design option not to take the white space into consideration. But please correct me if I'm wrong and let me know if there are any other solutions to handle this issue, like the scape character?

TOML
```
link = "https://alice.example.com"
email = "alice@example.com"
name = "Alice Wang"
```

MD
```
---
...
draft: false
authors: ["Alice Wang"]
...
---
```

BTW, another question is about how to use `lastmod` front matter as the example archetypes define `data` and `lastmod` with the same value.

Thanks! :)

**Comment by HEIGE-PCloud** [comment, marked as answer]
Hi

The white space in author name is not taken into consideration while implementing the author taxonomy. But I think it is very sensible to allow white space in author's name. I plan to refactor the author taxonomy in the near future and this feature will be taken into account.

For now, you probably can use the i18n feature to get around this issue.

```toml
link = "https://alice.example.com"
email = "alice@example.com"
name = "Alice"
[en]
    name = "Alice Wang"
```

This configuration may work, but I have not tested it on single language sites and "Alice" instead of "Alice Wang" might show up at some places.

For the second question, the `lastmod` front matter will change the "Updated on {{ time }}" text at the footer of each post. You can update it by yourself.

![image](https://user-images.githubusercontent.com/52968553/125162280-7ef33700-e1b9-11eb-80f2-04dbccc46db8.png)

Or you can set `enableGitInfo = true` in `config.toml` and configure the value for `gitRepo` and this value will be auto generated by Hugo.

Reference:
- https://hugodoit.pages.dev/theme-documentation-basics/#3-configuration
- https://gohugo.io/variables/git/

**Reply by zhaoyiluo** [reply]
Always appreciate your prompt and detailed reply!
